activiti-cloud-services-query/activiti-cloud-services-query-rest/src/main/resources/application.properties

```diff
@@ -19,4 +19,7 @@ keycloak.principal-attribute=preferred-username
 
 activiti.keycloak.admin-client-app=admin-cli
 activiti.keycloak.test-user=client
-activiti.keycloak.test-password=client
\ No newline at end of file
+activiti.keycloak.test-password=client
+
+# enable clean up endpoints
+activiti.enable-clean-up=true
```

Comment:
Actually, I think this is not going to work. As far as I remember this `application.properties` will be erased by the one coming from the end application, i.e. [activiti-cloud-query](https://github.com/Activiti/activiti-cloud-query/blob/master/src/main/resources/application.properties). We need to remove this `application.properties` from here. Can you create an issue for it, please?

Currently, the way we are using to load properties in the end application is having them in a dedicated properties file (similar to [this one](https://github.com/Activiti/activiti-api/blob/develop/activiti-api-model-shared-impl/src/main/resources/conf/rest-jackson-configuration.properties)) and the property file should be referenced by and auto-configuration (as [here](https://github.com/Activiti/activiti-api/blob/develop/activiti-api-model-shared-impl/src/main/java/org/activiti/api/runtime/conf/impl/CommonModelAutoConfiguration.java#L37))